src/commands/create.js

```diff
@@ -190,6 +191,19 @@ export default asyncCommand({
 			spinner.succeed('Done!\n');
 		}
 
+		if (argv.less || argv.sass || argv.stylus) {
+			let extension;
+
+			if (argv.less) extension = '.less';
+			if (argv.sass) extension = '.scss';
+			if (argv.stylus) extension = '.styl';
+
+			const cssFiles = await promisify(glob)(`${target}/src/**/*.css`);
```

Comment:
@rkostrzewski @lukeed i can change it to this one: 
```js
const cssFiles = await promisify(glob)(`${target}/**/*.css`, { ignore: `${target}/build/**/*` });
```
Just tested with `root` template and it worked.